### ISSUE TITLE
Nodeos Logging Documentation

### ISSUE THREAD

**Issue by deckb** [EOSIO/eos#5770]
I have not seen any good documentation for the logging.json and I believe that would be a very nice addition.

**Comment by fproulx-eoscanada** [comment]
Yes !!! It's very annoying to have that much noise in standard out by default.
